tests/10-vcdm2.js

```diff
@@ -608,26 +604,20 @@ describe('Validity Period', function() {
         this.test.link = `https://w3c.github.io/vc-data-model/#validity-period:~:text=If%20a%20validFrom%20value%20also%20exists%2C%20the%20validUntil%20value%20MUST%20express%20a%20datetime%20that%20is%20temporally%20the%20same%20or%20later%20than%20the%20datetime%20expressed%20by%20the%20validFrom%20value.`;
         const positiveTest = require(
           './input/credential-validUntil-validFrom-ok.json');
-        positiveTest.validFrom = createTimeStamp({skew: -2});
-        positiveTest.validUntil = createTimeStamp({skew: 2});
+        positiveTest.validFrom = createTimeStamp({skew: -1 * oneYear});
+        positiveTest.validUntil = createTimeStamp({skew: oneYear});
```

Comment:
The skew changed from 2 to 1 years. Not a big deal, I guess?

I do find the `-1 * oneYear` confusing...

Maybe just adding comments would be clearer than a variable that equals `1`?

Reply:
I felt the confusion was what the number represents as skew does now explicitly tell you that the skew is years. Perhaps it should be skewYears as the parameter to the function instead? That seems like an improvement and I can make it if needed.

Reply:
Agreed. That interface would be much more clear. Thanks.